Sources/SwiftRefactor/ExpandEditorPlaceholder.swift

```diff
@@ -137,12 +135,10 @@ struct ExpandSingleEditorPlaceholder: EditRefactoringProvider {
 /// foo(
 ///   closure1: <#T##(Int) -> String##(Int) -> String##(_ someInt: Int) -> String#>,
 ///   normalArg: <#T##Int#>,
-///   closure2: { ... }
-/// ) { someInt in
-///   <#T##String#>
-/// } closure2: { someInt in
-///   <#T##String#>
-/// }
+///   closure2: { ... },
+///   closure3: { <#someInt#> in <#T##String#> },
+///   closure4: { <#someInt#> in <#T##String#> }
+/// )
```

Comment:
Similar here, I think that the general `ExpandSingleEditorPlaceholder` refactoring should produce trailing closures, we just want to tweak the behavior from SourceKit-LSP. Maybe we need an option for that.